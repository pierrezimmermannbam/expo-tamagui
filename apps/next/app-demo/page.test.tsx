import { View } from 'react-native';
import { render } from '@testing-library/react';

test('render a View', () => {
  const { baseElement } = render(<View />);

  expect(baseElement).toMatchSnapshot();
});
