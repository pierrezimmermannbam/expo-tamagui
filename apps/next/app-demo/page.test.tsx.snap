// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a View 1`] = `
<body>
  <div>
    <div
      class="css-view-175oi2r"
    />
  </div>
</body>
`;
